metadata-ingestion/docs/sources/powerbi/powerbi_recipe.yml

```diff
@@ -16,6 +16,8 @@ source:
     client_secret: bar
     # Enable / Disable ingestion of user information for dashboards
     extract_ownership: true
+    # Enable / Disable ingestion of endorsements
+    extract_endorsements_to_tags: false
```

Comment:
we can keep it true by default

Reply:
Sure!

Reply:
As per John's comment, I'll keep this disabled by default.

Reply:
Agree